spec/lib/cds_importer/entity_mapper/base_mapper_spec.rb

```diff
@@ -229,14 +229,6 @@ def self.name
       it { is_expected.not_to be_destroy_operation }
     end
 
-    it_behaves_like 'an xml node and mapper that are not a destroy operation', 'D' do
```

Comment:
The destroy_operation?/whether the feature is enabled have been totally separated, now